data/test/macros/wml_unit_test_macros.cfg

```diff
@@ -88,6 +88,7 @@ Elvish Archer#endarg
                 name = "Alice"
                 type = {SIDE1_LEADER}
                 id=alice
+                random_traits=no
```

Comment:
Why is this line necessary? Alice and Bob already don't get random traits, so if this line is needed then it indicates an API change that's subtle but which will affect UMC.

Reply:
They don't get random traits? How do you know? I can remove it if it's redundant.

Reply:
I ran a test within the GUI (ran it with `--test`) and checked.

Reply:
Why do they not get random traits if they don't have this line…?